renovate.json

```diff
@@ -3,5 +3,8 @@
   "extends": ["local>cds-snc/renovate-config"],
   "lockFileMaintenance": {
     "schedule": ["before 6am on the first day of the month"]
+  },
+  "changelog": {
+    "template": "{{version}}\n\nReleased on: {{date}}"
```

Comment:
Can we move this to a separate PR so it's cleaner and easier to reference in the future? TY!